language/move-lang/functional-tests/tests/libra/libra/add_currency_to_system.move

```diff
@@ -0,0 +1,159 @@
+//! account: vivian, 1000000, 0, validator
```

Comment:
This is an example of adding a currency to the system using admin scripts, and making sure that it's fully operational. This wasn't possible previously since the writeset of the admin script would intersect with the write set of the `writeset_epilogue` at a `LibraConfig::NewEpochEvent`.